docs/content/en/docs/migrate/metrics-observe/_index.md

```diff
@@ -11,3 +11,106 @@ Information that is published here has been reviewed for technical accuracy
 but the format and content is still evolving.
 We hope you will contribute your experiences
 and questions that you have.
+
+The SLIs and SLOs used for Keptn v1 quality gates can be ported to
+KLT Keptn Metrics and Keptn Evaluations.
+
+Note that KLT includes additional observability features
+that are not included in Keptn v1 by default:
+
+* [Dora metrics](../../implementing/dora)
```

Comment:
`DORA` is an abbreviation for `DevOps Research Agency` not `Dora`.

Reply:
Fixed